lldb/source/Plugins/ExpressionParser/Swift/SwiftExpressionParser.cpp

```diff
@@ -1371,14 +1371,14 @@ static llvm::Expected<ParsedExpression> ParseAndImport(
   //        inserting them in.
   swift_ast_context.AddDebuggerClient(external_lookup);
 
-  if (expr_diagnostics.HasErrors())
```

Comment:
On second read of this, what do you think about making `GetErrorKindIfAny` an added method, instead of a changed method?

Then, `HasErrors` would be:

```cpp
bool SwiftASTContext::ScopedDiagnostics::HasErrors() const {
  return GetErrorKindIfAny() != std::nullopt;
}
```

The reason for this suggestion is that `HasErrors` reads better on the majority of these callsites, and `HasX` is a common convention.